### PR TITLE
CP-20221: Labels and Annotations

### PR DESCRIPTION
**Pull request by josephbarnett**
### Description

* Release notes for 1.0.0 Release
* Troubleshooting guide for Certificates Issues
* Bump version of insights-controller image

### Testing

N/A

### CODE REVIEW

**Review by adamrice32 — COMMENTED** [review]
Question: cert-manager is defaulted to enabled, and yet you state that most environments will already have this deployed. is it just me or is this contradictory?

**Review by teevans — APPROVED** [review]

**Comment by josephbarnett** [comment]
> Question: cert-manager is defaulted to enabled, and yet you state that most environments will already have this deployed. is it just me or is this contradictory?

Great question! I think in most "production" environments we've seen they already have a deployment. I think for easy of install we had this enabled, but maybe we should disable it? @dmepham would like your input here.

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+- **Using `cert-manager`**:
+  ```yaml
+  cert-manager:
+    enabled: true
+  insightsController:
+    cert-manager:
+      enabled: true
+  ```
+  
+- **Using `cloudzero-certificate` Chart**:
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:62]: This is the same configuration shown for using custom certificates -- is that OK?

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+  cert-manager:
+    enabled: false
+  insightsController:
+    certificate:
+      enabled: false
+    issuer:
+      enabled: false
+  ```
+
+- **Using Custom Certificates**:
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:73]: This is the same configuration shown for using cloudzero-certificate -- is that OK?

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+      enabled: false
+  ```
+
+### 2. Check Certificate Status
+
+#### If Using `cert-manager`:
+
+- **Verify `cert-manager` is Running**:
+  ```bash
+  kubectl get pods `-n <YOUR_NAMESPACE>`
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:90]:
```suggestion
  kubectl get pods -n <YOUR_NAMESPACE>
```

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+  ```
+  Ensure all `cert-manager` pods are in the `Running` state. Replace the `-n <YOUR_NAMESPACE>` with the cert-manager namespace. eg. `-n cert-manager`.
+  
+  eg.
+  ```bash
+  $kubectl get pods -n cert-manager
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:96]:
```suggestion
  $ kubectl get pods -n cert-manager
```

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+
+---
+
+## Additional Resources
+
+- [Kubernetes Admission Controllers Documentation](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+- [`cert-manager` Documentation](https://cert-manager.io/docs/)
+- [CloudZero Agent Helm Chart README](./README.md)
+- [ValidatingWebhookConfiguration Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#validatingwebhookconfiguration-v1-admissionregistration-k8s-io)
+- [Helm Troubleshooting Guide](https://helm.sh/docs/chart_best_practices/troubleshooting/)
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:363]: This URL is broken; should we link to `https://helm.sh/docs/faq/troubleshooting/` instead?

File: charts/cloudzero-agent/docs/cert-trouble-shooting.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+
+---
+
+If you continue to experience certificate-related issues after following this guide, consider reaching out to [CloudZero Support](https://cloudzero.com/support) or filing an issue on the [CloudZero Helm Charts GitHub repository](https://github.com/Cloudzero/cloudzero-charts/issues).
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/cert-trouble-shooting.md:367]:
```suggestion
If you continue to experience certificate-related issues after following this guide, consider reaching out to [support@cloudzero.com](mailto:support@cloudzero.com) or filing an issue on the [CloudZero Helm Charts GitHub repository](https://github.com/Cloudzero/cloudzero-charts/issues).
```

File: charts/cloudzero-agent/docs/releases/1.0.0.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+   >
+   > **Replace** `<YOUR_NAMESPACE>` with the namespace you used for your deployment.
+
+### Deprecations and Breaking Changes
+
+1. **`node-exporter` Deprecation**:
+   
+   The `node-exporter` has been deprecated and is no longer used.
+
+2. **`kube-state-metrics` Configuration Changes**:
```
Comment (beckilee) [inline charts/cloudzero-agent/docs/releases/1.0.0.md:104]: Did we rename this `cloudzero-state-metrics`?

**Review by beckilee — COMMENTED** [review]
Just a few editorial suggestions, but otherwise looks good!

File: charts/cloudzero-agent/docs/releases/1.0.0.md (hunk 7 above)
Comment (josephbarnett) [reply charts/cloudzero-agent/docs/releases/1.0.0.md:104]: Good eye! We renamed the service instance name (think alias for the actual name) which shouldn't be conflicting. However we do deploy it by default and name it `cloudzero-state-metrics`.  I think that should be captured!

**Pull request merged**